src/main/webapp/WEB-INF/templates/views/privileged/my_flat.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/head.html :: head"></head>
+<body>
+<div class="container-fluid">
+    <div class="container">
+        <span th:if="${RightToSeeFlat}">He has right to see the flat</span>
```

Comment:
tu by som nedala He ale User alebo He/She alebo nieco podobne, nie je to ale podstatne

Reply:
Toto je docasne, aby ste videli, ze tam nieco realne je co sa tyka toho projektu to cele bude nahradene predsa tymi meraniami a pod